test/public/defaults.js

```diff
@@ -145,17 +145,17 @@ module.exports.waitForTimeout = waitForTimeout;
  *
  * @param {puppeteer.Page} page - The puppeteer page where the table is located.
  * @param {number} expectedSize - The expected number of table rows, excluding rows marked as loading or empty.
+ * @param {string} [operator = '==='] comparison operator
  * @return {Promise<void>} Resolves once the expected number of rows is met, or the timeout is reached.
  */
-const waitForTableToLength = async (page, expectedSize) => {
+const waitForTableLength = async (page, expectedSize, operator = '===') => {
```

Comment:
No. In tests you must know exactly what to expect.
`` makes the code harder to understand, have no autocomplete... I strongly disagree with this.
Also, you should refrain from changing the behavior of functions. Function named waitForTableToLength should do what it means (if you need to rename it it's even more obvious that you should not do it), if you need to extend the behavior you should simply create a new one.

Reply:
applied